### PR TITLE
Improved power limiter documentation

### PR DESCRIPTION
**Pull request by MalteSchm**
I drafted some documentation that describes the power limiter a) for other users and b) to offload me mentally when looking at the bugfix for https://github.com/helgeerbe/OpenDTU-OnBattery/issues/193

@helgeerbe I hope that I have understood it now but it would be good if you can crosscheck

### CODE REVIEW

File: README_onBattery.md [hunk 1]
```diff
@@ -92,7 +92,25 @@ Several screenshots of the frontend can be found here: [Screenshots](docs/screen
...
+
+* Solar Passthrough is off
+  * When using this strategy the inverter is steered such that the currently consumed power (as provided by the power meter) is compensated for. This is done as long as the battery charge state is above the stop threshold. The inverter is turned off if the battery reaches the limit set by stop threshold.
+* Solar Passthrough is on and the battery drain strategy is empty at night
+  * When using this strategy the inverter is steered such that the currently consumed power (as provided by the power meter) is compensated for. It will use only solar power if solar is available (i.e. daytime) and battery power at night. When operating with solar power the inverter power limit is constrained to the input solar power and the power meter value so that battery discharge is avoided. At night the battery is discharged provided that the battery charge state is above the limit set by stop threshold. To switch between day / night time 20W solar power from the Victron MPPT Solar Charger are required.
```
Comment (helgeerbe) [inline README_onBattery.md:102]: If voltage is above start threshold, battery will be used until voltage is below start threshold again. (don't waste usable energy)

**Review by helgeerbe — COMMENTED** [review]
I think also the start condition is missing. This is part of the problem in #193.

If inverter is switched off, due to low voltage and the battery recovers, the inverter is switched on again...
Old implementation kept the inverter switched off, until start condition was reached:
* startThreshhold is reached
* if solarPassthrough is activated, solar power could be consumed

**Comment by MalteSchm** [comment]
@helgeerbe 
I reworked the text once more to include the start threshold condition and was able to leverage some similarities 

@madmartin
I tried to rebase and force push now to end up with a single commit. I hope that this did the trick.

**Comment by helgeerbe** [comment]
@MalteSchm Yes, your statement is correct!

**Comment by MalteSchm** [comment]
@helgeerbe 
I updated the text and consider this ready for merging now

**Review by helgeerbe — APPROVED** [review]

**Pull request merged**